content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -51,6 +51,21 @@ may [fail](https://github.com/kubernetes/kubeadm/issues/31).
 If you have more than one network adapter, and your Kubernetes components are not reachable on the default
 route, we recommend you add IP route(s) so Kubernetes cluster addresses go via the appropriate adapter.
 
+## Ensure iptables does not use the nf_tables backend
+
+On Debian 10 (Buster) and other recent distributions, the `iptables` binary uses the newer `nf_tables`
+backend by default. This backend is not compatible with current Kubernetes releases: it causes duplicated
+firewall rules and breaks `kube-proxy`.
```

Comment:
Maybe "not compatible with the current kubeadm release"? The "breaks kube-proxy" thing is a function of having a mismatch between components, not of using nft in general.

Reply:
The issue is not with kubeadm but with kubelet and kube-proxy that both call out to `iptables`. Kubeadm is just the most commonly used tool to build clusters, so it will be a common avenue for hitting this problem.

Furthermore, https://github.com/kubernetes/kubernetes/issues/82361#issuecomment-529668136 mentions that iptables-nft in Debian 10 is broken in a way that Kubernetes (kubelet) can't use it. To me, this sounds like recommending people to avoid nft is probably the best thing we can do for now.

Reply:
Maybe "iptables v1.8.2 and below is not compatible..." - rather than a blanket statement.

Reply:
Please correct me if I’m wrong, but as far as I know, host OS’es using iptables v1.8.3 are still not compatible with the kube-proxy image that is installed by kubeadm (which has iptables tooling that doesn’t know about nft at all).

Reply:
Ah. Well, if that's the case, then yes, you're right - the host needs to match whichever the kube-proxy has. Which could theoretically change at any time :-)